src/renderer/components/+apps-releases/release.store.ts

```diff
@@ -78,9 +78,15 @@ export class ReleaseStore extends ItemStore<HelmRelease> {
   }
 
   async loadItems(namespaces: string[]) {
-    return Promise
-      .all(namespaces.map(namespace => helmReleasesApi.list(namespace)))
-      .then(items => items.flat());
+    const isLoadingAll = namespaceStore.allowedNamespaces.every(ns => namespaces.includes(ns));
+
+    if (isLoadingAll) {
```

Comment:
This is to prevent a similar regression that I found in `kube-object-store.ts`. See #2142 

```suggestion
    if (isLoadingAll && namespaceStore.context.cluster.accessibleNamesapces.length === 0) {
```

Reply:
Thanks for spotting! Fixed.